### DISCUSSION TITLE
Getting a positive hit on VirusTotal

### DISCUSSION THREAD

**Discussion by ghost** [tranxuanthang/lrcget#43] [General]
Hi,

Just to flag, you're getting a positive flag on VirusTotal. It may be nothing but I thought I ought to let you know.

https://www.virustotal.com/gui/file/7e6ff9b279d7a0083b75faabe98d7650de383cae5e4e2fab1ff69d6bcc2efd36?nocache=1

**Comment by tranxuanthang** [comment]
It should be a false positive. GitHub had removed this repository once in the past, because the prebuilt binaries for Windows requires UAC for installation, so this could be a reason.
